src/components/MembersPanel.tsx

```diff
@@ -368,9 +368,31 @@ const MembersPanel = ({ members, setMembers }: MembersPanelProps) => {
       );
    }
 
+   function totalMembers(): number {
+      return (
+         members.length
+      );
+   }
```

Comment:
since this function is very simple, we can actually remove it and directly use {members.length}